resources/backend/sql_transactions.md

```diff
@@ -0,0 +1,9 @@
+## What is a SQL transaction?
+
+A SQL transaction is a way to use commands in the database, but not have them actually affect a main database -- sort of a testing case.
```

Comment:
I think we need a little more than this. Talk about the general concept of what it does. When we talk about SQL transactions, the ACID concept often comes up - tho for us, the A (Atomicity) is the most important concept. I like the ACID explanation and the bank analogy provided in this stack overflow: https://stackoverflow.com/questions/974596/what-is-a-database-transaction